### PR TITLE
#46 added  sidebar nodes

### PR DESCRIPTION
**Pull request by emekauja**
added sidebar nodes for the icons and label

### CODE REVIEW

**Comment by emekauja** [comment]
Screenshot 2022-11-02 at 3 28 08 PM

File: src/components/views/sidebar/nodes.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+} from './icons';
+
+export const sideBarNodes = [
+  {
+    id: 'targetMechanism',
+    name: 'Target',
+    icon: <TargetIcon />,
+  },
+  {
+    id: 'targetMechanism',
```
Comment (ddelpiano) [inline src/components/views/sidebar/nodes.js]: typo

**Review by ddelpiano — CHANGES_REQUESTED** [review]
@emekauja just a minor typo and then this can be approved and merged. Thx

File: src/components/views/sidebar/nodes.js (hunk 1 above)
Comment (emekauja) [reply src/components/views/sidebar/nodes.js]: fixed

**Review by ddelpiano — APPROVED** [review]

**Review by ddelpiano — APPROVED** [review]

**Pull request merged**